backend/migrate/migrations/scripts/migration_15.sql

```diff
@@ -0,0 +1,17 @@
+CREATE OR REPLACE VIEW view_index_all_stats
+AS
+SELECT stats.monitored_prefixes, stats.configured_prefixes,
+ 	(SELECT count(*) Total_Hijacks FROM hijacks WHERE key is not NULL),
```

Comment:
if the cameltoe is ditched by postgres, fix it on the frontend side and keep lower lettering here

Reply:
Fixed.